### PR TITLE
Refactor move

### PR DESCRIPTION
**Pull request by patrixr**
This PR goes hand in hand with this [backend change](https://github.com/crossroads/api.goodcity/pull/854) (read first for context)

We want to start using the new Order fulfilment "operations" (as they've been called on the api).
That will allow a smooth(er) transition to a stockit free world later on since those have been implemented with that in mind.

As a result, here are the changes that occurred : 

- Implementing the proper "move" workflow, which assumes multi-location and partial quantity is possible. (modifying the quantity is locked for the time being, but fully functional)
- removing any code that refers to old endpoints or previous partial quantity implementation

### CODE REVIEW

File: app/mixins/move_actions.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+    return this.get("locationService").userPickLocation({
+      headerText: text,
+      presetLocations: presetLocations
+    });
+  },
+
+  quantityAtSource() {
+    const pkg = this.get("moveTarget");
+    const source = this.get("moveFrom");
```
Comment (namrataukirde) [inline app/mixins/move_actions.js]: How about naming it as `sourceLocation`?

File: app/services/location-service.js [hunk 2]
```diff
@@ -21,6 +23,20 @@ export default ApiBaseService.extend(NavigationAwareness, {
     this.getWithDefault("onLocationSelected", _.noop)(null);
   },
 
+  async movePackage(pkg, opts = {}) {
+    const { from, to, quantity } = opts;
+
+    const payload = await this.PUT(`/packages/${pkg.get("id")}/move`, {
```
Comment (namrataukirde) [inline app/services/location-service.js]: Is there any scenario where we need to show an error to users if any move operation fails?

File: app/mixins/move_actions.js (hunk 1 above)
Comment (patrixr) [reply app/mixins/move_actions.js]: mostly for consistency with the backend (more aesthetically pleasing too imo)

File: app/services/location-service.js (hunk 2 above)
Comment (patrixr) [reply app/services/location-service.js]: There are, but not in the service. An exception coming from the service is a perfectly valid outcome.

Whoever calls the service has to handle the error, that would usually be the page.
Right now it's letting the default error handler show the popup, but I do have an improvement in mind regarding that : )

**Pull request merged**